### ISSUE TITLE
admin ui does not indicate bad syntax

### ISSUE THREAD

**Issue by jhorwit2** [eBay/fabio#84]
It appears that you can save invalid overrides. For example,

`route weight hello_world/ weight 0.05 tas "test`

which in fabio's output has
 `2016/04/18 14:56:37 [WARN] route: line 3: syntax error in route weight hello_world/ weight 0.05 tas "test`

**Comment by magiconair** [comment]
Yes, that is true - unfortunately. The parser will not allow it but the rest of the code assumes to get only valid data and I do not pass the errors along. Should not be too difficult but I am busy with something else right now.
